ansible/artifacts/sunbird/login/resources/js/login.js

```diff
@@ -437,7 +437,7 @@ var handleSsoEvent  = () => {
 		const sessionUrlObj = new URL(sessionUrl);
 		if (redirect_uri) {
 			const redirect_uriLocation = new URL(redirect_uri);
-			if (client_id === 'android') {
+			if (client_id === 'android' ||  client_id === 'desktop') {
```

Comment:
we don't have any client in keycloak named desktop

Reply:
@HarishGangula @Nik720 is this new keycloak client reviewed by DC for the authentication and authorization along with valid redirect routes?

Reply:
@keshavprasadms had created a new KC client for desktop for staging

Reply:
Thats only for testing your code. The design has to be signed off from DC.